### DISCUSSION TITLE
how to run the core project?

### DISCUSSION THREAD

**Discussion by hparsafar** [WalletConnect/walletconnect-monorepo#5220] [Developer Support]
I don't know how to start the relay server. Can you help me?
I don't want to run checks for packages.
I want to start each project, only the sign-client has a Docker file.
Where to put my project ID in the project?

thank you

**Comment by linear** [comment]
<p><a href="https://linear.app/walletconnect/issue/CR-484/how-to-run-the-core-project">CR-484 how to run the core project?</a></p>

**Comment by glitch-txs** [comment]
Hello, the relay server is currently private, we will open source it soon.
